lib/index.js

```diff
@@ -124,7 +124,7 @@ api.set = async ({
     const {ttl} = cfg.nonce.defaults;
 
     // generate new token
-    challenge = await generateNonce();
+    challenge = await generateNonce({type: entryStyle});
```

Comment:
If we do the above, with type-specific options, we could use `entryStyle` here as the param name (instead of `type`) and just pass the options right through.